stable/graylog/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: graylog
+home: https://www.graylog.org
+version: 0.1.0
```

Comment:
I think we want to go to incubator.
Especially if version is 0.1.0

Reply:
Sure, I will move to `incubator`